docs/DevDocs/GettingStarted.md

```diff
@@ -0,0 +1,156 @@
+# Getting Started
+
+This document gives an overview of the code contained in this monorepo and the recommended development setup.
+
+## Develop with Docker (recommended)
+This is the simplest configuration for developers to start with.
+
+1. Make a copy of `template.env` and call it `.env`.
+2. Set the environment variables in `.env`.
+3. Run `docker compose up` to start the Django development server and Celery worker, plus all backing services
+   like PostGIS, Redis, RabbitMQ, etc.
+4. Run `docker compose run --rm django poetry run django-admin migrate` to apply database migrations.
+5. Run `docker compose run --rm django poetry run django-admin loaddata lookups` to initialize your database with required data.
+6. Optionally, populate your database with test data by running `docker compose run --rm django poetry run django-admin loaddata testdata`
```

Comment:
This testdata has been outdated for a long time, i think we should exclude it from the docs.
```suggestion
```